### ISSUE TITLE
Naming: "The Carpentries"?

### ISSUE THREAD

**Issue by k8hertweck** [carpentries/2017Merger#10]
A discussion regarding the names we're applying to the umbrella organization started in the comments section of this [blog post](https://software-carpentry.org/blog/2017/09/merger.html#comment-3530592194). The basic concern is that "The Carpentries" is not immediately recognizable as a software/data/computation related organization. Specific posts are as follows:

From Dena Strong:
I love the idea of merging the organizations, but the name worries me - the word "Carpentries" by itself is terrible as a findable brand because it doesn't retain either of the words that makes it distinguishable from traditional wood-based carpentry in search engines. It's a name with internal singificance to people in the know that loses the keywords involved n its external significance to people we're trying to reach. I'd be fine with either Software Carpentry or Data Carpentry (or conceivably Software and Data Carpentry) as a joined name, but I can't imagine "Carpentries" being a word that would mean "Learn how to program" to people who didn't already know the history of the name.

My response:
Thanks for this comment, and your point is well taken! "The Carpentries" has been our operational/working term for the umbrella organization, in part because we are planning for this restructured organization to be able to expand to other Carpentry groups in the future. Perhaps a more general term like "Programming Carpentries" or "Coding Carpentries" would be inclusive enough, while also informative for branding?

From Jeremy Zucker:
How about Digital Carpentries as the umbrella organization, with software and data as the two underlying digital instantiations?
On a philosophical/historical aside, data and software are indistinguishable to a "stored program digital computer." http://wiki.c2.com/?DataAndCodeAreTheSameThing 
Thus it always seemed to me quite fitting that software and data carpentry would eventually merge into digital carpentries.

**Comment by ChristinaLK** [comment]
Computing Carpentries?

**Comment by kbroman** [comment]
I agree with Dena's comments about "Carpentry" vs "Carpentries" and would favor either "Data and Software Carpentry" or "Software and Data Carpentry".

**Comment by npch** [comment]
I wonder if the right way to do this is simply to continue with our well-understood and well-recognised brands for lesson sets (Software Carpentry, Data Carpentry, Library Carpentry etc) and then just pick a name for the umbrella organisation which is easy to differentiate and represents our vision.

Using the example of Apache, you have the Apache Software Foundation and then the projects (Apache Spark, Apache Solr etc).

So you could have "The Software and Data Carpentry Foundation" or "The Digital Carpentry Foundation" or even "The Fabrilis Foundation: home of Software and Data Carpentry" _(I'm sure an actual Latin scholar will be able to correct my usage)_ and then each lesson org would be of the form of "X Carpentry". The name of the umbrella foundation doesn't matter as much as ensuring that people understand that it is the umbrella organisation for doing this kind of teaching.

I think an important part of this discussion is what we as a community collectively buy into as the vision and values of the movement as a whole. A lot of us have come from Software Carpentry, so there's a bias towards coding but we now have a wider set of members for whom the movement is around a particular format and ethos of teaching skills to researchers (potentially resulting in things like Author Carpentry).

**Comment by tracykteal** [comment]
Definitely we want to continue with the brands for lessons (Software Carpentry, Data Carpentry, Library Carpentry) and we need to decide the name for the umbrella organization. It's a good point that the 'Carpentries' isn't descriptive enough to make sure people understand the vision/goals of the organization. So, there does need to be a name that is more descriptive and provides an appropriate home name for Lesson Carpentries. It also means that we need to be clear on the vision of the organization as we think of a name. Teaching skills to researchers can encompass something different than a focus on computing skills, so it's an important point for discussion.

**Comment by gvwilson** [comment]
I like "Research Carpentry" (since everyone we're intending to teach does research of some kind) or "Lesson Carpentry" (which puts more of an emphasis on our content). As far as I can tell, neither name has been trademarked in the US or Canada, which I think ought to be a must-have requirement for any name - I very much doubt we'd get a trademark on "The Carpentries".

**Comment by kcranston** [comment]
I like "Research Carpentry". I've been assuming that "The Carpentries" is a placeholder and that we would come up with a better name at some point. Thanks to @k8hertweck for explicitly starting the conversation. I think that using "Software and Data" in the name of the umbrella org does not leave enough room for new lesson organizations, and permanently confers a special status to the two originating organizations .

**Comment by ethanwhite** [comment]
I like "Research Carpentry"/"The Research Carpentry Foundation" as well. RCF has a solid feel as an acronym as well.

**Comment by k8hertweck** [comment]
At a meeting of members from both DC and SWC governance today, we discussed possible names for the merged organization. We are considering the name "Digital Carpentries." Other names considered included:

- The Carpentries (although it lacks coherence in terms of our approach to data and programming, and some participants noted a dislike for a prefix of "The)
- Research Carpentry (although this name is exclusionary to parts of our communities who are not focused on research, like libraries)
- Software and Data Carpentry (although accurate and inclusive, would promote the appearance of the previous organizations of SWC and DC as potentially more important/foundational than other lessons)

While there is some concern that Digital Carpentries is too nebulous a descriptor for activities in which our community engages, it does reflect other nomenclature for programmatic approaches to problem solving (such as digital humanities).

We are interested in how this sounds to others in the community. (@mkuzak)
